css/properties/-moz-context-properties.json

```diff
@@ -27,14 +27,8 @@
             },
             "firefox_android": {
               "version_added": "55",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "svg.context-properties.content.enabled",
-                  "value_to_set": "true"
-                }
-              ],
-              "notes": "With the preference set to <code>false</code>, the property still works with SVGs via <code>chrome://</code> or <code>resource://</code> URLs"
+              "partial_implementation": true,
+              "notes": "The property only works with SVGs via <code>chrome://</code> or <code>resource://</code> URLs."
```

Comment:
Why this note and partial implementation? Seems like the feature works in some non-web-exposed contexts despite the flag, but with the flag removed is there anything to say still?

Reply:
I guess it's not helpful to say that a feature is supported only in internal contexts, so we should probably just set this to "false"?

Reply:
I think so, yes. But can you search for other comments involving chrome:// or similar and see if there's a lot of them? If there are I think we should remove them all in one PR and have a clear justification, for future reference.

Reply:
It appears this is the only instance containing `resource://` or `chrome://` (besides references to `chrome://flags` for Google Chrome) -- removal inbound!

Reply:
I opened #15870 to remove this property altogether.